### DISCUSSION TITLE
Make this repo available on registry.bazel.build?

### DISCUSSION THREAD

**Discussion by ztl8702** [thesayyn/protoc-gen-ts#189] [Ideas]
So that its `ts_proto_library` rule can be more easily consumed

**Comment by thesayyn** [comment]
I have no objections to the idea. However, with the limited time I got available, I can't make this happen anytime soon.

**Reply by ztl8702** [reply]
No problem sir. I can take a stab when I have time.

**Reply by thesayyn** [reply]
That'd be great. I believe this should be an issue. any objections to converting to an issue?

**Reply by ztl8702** [reply]
no objection. Go ahead

**Comment by thesayyn** [comment]
Closing this as we no longer expose this repo as a bazel rule.
